src/highOrderComponents/AuthenicatedRoute.tsx

```diff
@@ -3,46 +3,69 @@ import React, { Component } from 'react'
 import { Route, Redirect, RouteProps } from 'react-router-dom'
 
 import { connect } from 'react-redux'
-import { ReducersMapObject } from 'redux'
+import { ReducersMapObject, Reducer } from 'redux'
+import { logOut } from '../actions/authentication'
 
 export interface IProps {
-  path: string;
-  user: boolean;
+  user?: boolean | null;
   component: React.ComponentType<any>;
   pending: boolean;
+  path: string;
+}
+
+//add in logout to authenitacted routes
+interface StateProps {
+  authentication: Reducer
 }
 
+interface DispatchProps {
+  logOut: () => void
+}
+
+type Props = StateProps & DispatchProps & IProps
+
+class AuthenticatedRoute extends Component<Props, RouteProps> {
 
-class AuthenticatedRoute extends Component<IProps, RouteProps> {
   render() {
-    const { pending, user, path,  component } = this.props
-    // TODO: Use this?
-    const authentication = {
-      pending,
-      user
-    }
-  
-    if(pending && !user){
+    console.log("In authenitacted Route component", this.props)
```

Comment:
Can you go through this file and remove any console.logs used for debugging? Maybe just keep one that logs "logged in" when the login works and "signed up" when the signup works (or something similar)?